VoodooInput/VoodooInputSimulator/VoodooInputSimulatorDevice.cpp

```diff
@@ -218,17 +218,10 @@ void VoodooInputSimulatorDevice::constructReportGated(const VoodooInputEvent& mu
     else
         input_report.TouchActive = 0x2;
     
-    int total_report_len = (9 * multitouch_event.contact_count) + 12;
-    IOBufferMemoryDescriptor* buffer_report = IOBufferMemoryDescriptor::inTaskWithOptions(kernel_task, 0, total_report_len);
-
     if (!is_error_input_active) {
-      buffer_report->writeBytes(0, &input_report, total_report_len);
-      handleReport(buffer_report, kIOHIDReportTypeInput);
+        handleReport(memDesc, kIOHIDReportTypeInput);
```

Comment:
should set the buffer length to `total_report_len` before invoking `handleReport`.

Reply:
Is this still relevant?